examples/DateRangePickerWrapper.jsx

```diff
@@ -17,6 +17,7 @@ const propTypes = {
   autoFocusEndDate: PropTypes.bool,
   initialStartDate: momentPropTypes.momentObj,
   initialEndDate: momentPropTypes.momentObj,
+  isRTL: PropTypes.bool,
```

Comment:
I don't think this needs to go here since it probably should be added to DateRangePickerShape